must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -444,13 +445,18 @@ CMDS
         crash_collection &
         pids_log+=($!)
         journal_collection &
-        pids_log+=($!)
+        pids_journal+=($!)
```

Comment:
I'm not following the purpose of introducing `pids_journal` array in addition to existing `pids_log` array.  Aren't the respective processes waited for in exactly the same place?

Reply:
So we have some pid's limit, due to more volume collection, the pids reach their limit and logs are not getting collected further. Hence, we need to introduce a separate pid for journal logs.

Reply:
I don't see how this changes anything wrt. any potential limit (not sure what are referring to exactly).  The same number of processes is launched, it's just that instead of adding all their PIDs to the same array, you are adding `journal_collection` PID to the new array and then almost immediately wait for it.